lib/ui/toolbar.js

```diff
@@ -237,6 +238,7 @@ export default class Toolbar {
 						</Select>
 
 						<Button icon="gear" flat="true" disabled={this.state.disableUI || !this.state.codeSigningAvailable} click={this.expandButtonClicked.bind(this)} />
+						<Button icon="eye" flat="true" coDisabled={this.state.disableLiveView} disabled={this.state.disableUI || this.state.buildCommand !== 'run'} click={this.liveViewButtonClicked.bind(this)} />
```

Comment:
I think this button should be available when using custom builds? So maybe `!(this.state.buildCommand === 'run' || this.state.buildCommand === 'custom')`